docker_base_zulujdk_11_sbt/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM openjdk:11.0.1-jdk
```

Comment:
@pjfanning all gitlab images need to start with 
`FROM docker:latest` as per https://github.com/kontainers/docker_snipets/blob/master/docker_base_openjdk_8_sbt/Dockerfile#L1

The reason is that we are using docker commands in the images and so require a feature call `docker in docker`. Thats only available  from that base image. 

This base image has parctically nothing in it and will require installing everything (including the jre)

Normally what I do is to copy the dockerFile content (in this case the content of `FROM openjdk:11.0.1-jdk`) into this dockefile.